src/scss/_sofie-color-vars.scss

```diff
@@ -7,11 +7,13 @@ body {
   --dark-gray-color: #777777;
   --black-color: #000000;
   --black-color-with-alpha: #000000aa;
+  --hover-scrub-overlay-color-rgb: 0, 0, 0;
```

Comment:
What is the reason for this variable and can be HEX format and can use (`--black-color: #000000`)?

Reply:
If we were to extract each hex value for each gradient step into a separate variable, then we would get a lot, targeting very specific usecases . By doing this, we can append the alpha channel with the `rgba()` to the variable where needed. We haven't been able to find a way to use hex values, except for `color-mix` which seems to not be supported in the current versions of Chrome that we are running in production.